plugins/generic/pln/classes/DepositPackage.inc.php

```diff
@@ -192,9 +192,36 @@ function generateAtomDocument() {
 				$pkpDetails->setAttribute('checksumValue', md5_file($packageFile));
 				break;
 		}
-
+		
 		$entry->appendChild($pkpDetails);
 		$atom->appendChild($entry);
+		
+		$license = $atom->createElementNS('http://pkp.sfu.ca/SWORD', 'license');
+		$license->appendChild($atom->createElementNS('http://pkp.sfu.ca/SWORD', 'openAccessPolicy', $journal->getLocalizedSetting('openAccessPolicy')));
```

Comment:
I'd recommend getting the setting in the journal's primary locale, rather than the current locale. I don't know OTOH what the execution path to trigger this looks like, but if it's piggy-backed on a user request, it may result in the license information getting picked depending on the current user's language, which would be weird. (Possibly elsewhere as well)